### PR TITLE
Filing UI update the dashboard/12093

### PR DESCRIPTION
**Pull request by meganawong**
*Issue #:* /bcgov/entity#12093

*Description of changes:*
* updates the ledger entry for Put Back On and Administrative Dissolution Filings in Recent Filing History List

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by shabeeb-aot — APPROVED** [review]

File: src/components/Dashboard/FilingHistoryList.vue [hunk 1]
```diff
@@ -46,7 +46,9 @@
 
                 <!-- is this a Staff Only filing? -->
                 <div v-if="filing.isTypeStaff" class="item-header__subtitle">
-                  <FiledLabel :filing="filing" />
+                  <span v-if="filing.displayName === 'Correction - Put Back On' ||
+                  filing.displayName === 'Administrative Dissolution'">FILED<FiledLabel :filing="filing" /></span>
```
Comment (shabeeb-aot) [inline src/components/Dashboard/FilingHistoryList.vue:50]: can we use something else to check rather than checking the string? maybe name

Comment (meganawong) [reply src/components/Dashboard/FilingHistoryList.vue]: fixed.

**Review by shabeeb-aot — APPROVED** [review]

File: src/components/Dashboard/FilingHistoryList/FiledLabel.vue [hunk 2]
```diff
@@ -23,5 +29,9 @@ import { HistoryItemIF } from '@/interfaces'
 export default class FiledLabel extends Vue {
   @Prop({ required: true })
   readonly filing: HistoryItemIF
+
+  private get putBackOnOrAdminDissolution (): boolean {
+    return this.filing.name === 'putBackOn' || this.filing.name === 'administrativeDissolution'
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/FiledLabel.vue:34]: Use the enums instead of hard-coded strings.

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/FiledLabel.vue:34]: Better yet, aren't there getters or mixins for this?

**Review by severinbeauvais — COMMENTED** [review]
It's OK that this got merged already, but please create another PR to clean this up per my comments.

**Pull request merged**